### PR TITLE
Add: Korean language support

### PR DESCRIPTION
**Pull request by JefferyHus**

### CODE REVIEW

File: source/jsCalendar.lang.kr.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
+ * jsCalendar language extension
+ * Add French Language support
```
Comment (GramThanos) [inline source/jsCalendar.lang.kr.js]: You forgot the language name

File: source/jsCalendar.lang.kr.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ * Translator: Grammatopoulos Athanasios-Vasileios (gramthanos@github)
```
Comment (GramThanos) [inline source/jsCalendar.lang.kr.js]: Your name here too 😄

**Comment by GramThanos** [comment]
Also, based on [ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) the correct code is `ko`, `kr` is for `Kanuri`.

File: source/jsCalendar.lang.kr.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                // Add language to load list
+                window.jsCalendar_language2load.push(language);
+            }
+        };
+    }
+
+    // Add a new language
+    jsCalendar.addLanguage({
+        // Language code
+        code : 'kr',
```
Comment (GramThanos) [inline source/jsCalendar.lang.kr.js]: Wrong code for Korean. Change it to `ko` (the file name too).

**Comment by JefferyHus** [comment]
Done. Thanks for your comments :)

**Comment by GramThanos** [comment]
Thank you for your contributions! 😃

**Pull request merged**